chart/README.md

```diff
@@ -0,0 +1,102 @@
+# aws-event-sources
+
+Installs Triggermesh AWS event sources controller for knative. The following event sources are currently supported by the controller
+
+- Amazon CodeCommit (`kind: AWSCodeCommitSource`)
+- Amazon Cognito Identity (`kind: AWSCognitoIdentitySource`)
+- Amazon Cognito UserPool (`kind: AWSCognitoUserPoolSource`)
+- Amazon DynamoDB (`kind: AWSDynamoDBSource`)
+- Amazon Kinesis (`kind: AWSKinesisSource`)
+- Amazon Simple Notification Service (`kind: AWSSNSSource`)
+- Amazon Simple Queue Service (`kind: AWSSQSSource`)
+
+Refer to https://github.com/triggermesh/aws-event-sources/tree/master/config/samples for examples that make use of the controller.
+
+## TL;DR;
+
+```console
+$ helm install triggermesh/aws-event-sources
+```
+
+For support and feedback contact us as https://github.com/triggermesh/aws-event-sources/issues/new.
+
+## Introduction
+
+This chart installs the [aws-event-sources](https://github.com/triggermesh/aws-event-sources) controller on a Kubernetes cluster.
+
+## Prerequisites
+  - Kubernetes 1.16+ with Beta APIs
+  - Helm 3.0+
+  - Knative v0.14+
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm install --name my-release triggermesh/aws-event-sources
```

Comment:
you need to be outside the directory ? This assumes you cloned the repo correct ?

Reply:
Does this create the namespace and the CRD properly, meaning in the correct order ?

Reply:
>  you need to be outside the directory ? This assumes you cloned the repo correct ?

I've update the CI to publish the chart to https://storage.googleapis.com/triggermesh-charts and updated the instructions to add the repo to helm before installing the chart.

> Does this create the namespace and the CRD properly, meaning in the correct order ?

The namespace is not created by the chart. User needs to specify the namespace he/she wants to install the controller in using the `--namespace` argument to `helm install` and optionally `--create-namespace` if the namespace does not already exist.

Helm will probably create the namespace first when `--create-namespace` is specified, the CRDS's will be created before the templates are evaluated.